Sources/Subjects/ReplaySubject.swift

```diff
@@ -41,17 +41,17 @@ public final class ReplaySubject<Output, Failure: Error>: Subject {
         let subscriptions: [Subscription<AnySubscriber<Output, Failure>>]
 
         do {
-          lock.lock()
-          defer { lock.unlock() }
+            lock.lock()
```

Comment:
can you undo all of the re-formatting you did here please?

Reply:
I can do that. But it seems you are using Indent 4 everywhere and if you look into this file in the main branch, you will see that just this `do` scope has Indent 2. Let me know if that is what you want and I can change it back ;)